vue-frontend/tailwind.config.cjs

```diff
@@ -35,17 +37,30 @@ module.exports = {
           400: "rgb(141, 138, 138)",
         },
       },
+      borderRadius: {
+        "x-lg": "20px",
+      },
       borderWidth: {
         1: "1px",
       },
       boxShadow: {
         "3xl": "0px 4px 45px rgba(0, 0, 0, 0.1)",
+        "4xl": "2px 1000px 1px #3d3d3d inset",
       },
+
       lineHeight: {
         11: "2.75rem",
         17: "4.25rem",
         23: "5.75rem",
       },
+
+      fontFamily: {
```

Comment:
fontFamily is all ready define ??

Reply:
We have to place it here also, only then the new font will apply.